src/zeit/content/cp/interfaces.py

```diff
@@ -273,10 +273,11 @@ def getTitle(self, value):
 
 class QueryTypeSource(SimpleDictSource):
 
-    values = collections.OrderedDict((
-        ('channels', _('query-type-channel')),
-        ('serie', _('query-type-serie')),
-    ))
+    values = collections.OrderedDict((x, _('query-type-' + x)) for x in [
+        'channels',
+        'serie',
+        'product',
+    ])
```

Comment:
Hehe, bis zum ersten, wo das Muster nicht paßt… 😄

Btw, ich find ja die dictionary comprehension (oder heißen die dann "comprehension dictionaries"?) sehr schön zu lesen, also in dem Fall wär's dann sowas wie:
```python
values = collections.OrderedDict({x: _('query-type-' + x) for x in (
    'channels',
    'serie',
    'product',
)})
```

Reply:
Haha, muß ich doch im Urin gehabt haben oder so…  aber eigentlich: echt jetzt?  Das findet die sonst nich?

Reply:
Na der i18nextract ist stumpfes Text-Munging, der gräbt halt die Dateien durch -- da wird aber kein Python ausgeführt. 0